src/utils/fetchBoards.ts

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+import Board from "../interfaces/Board";
+
+async function fetchBoards(
+    setBoards: React.Dispatch<React.SetStateAction<Board[]>>,
+    backendUrl: string
+) {
+    const response = await axios.get(backendUrl + "boards");
+
+    setBoards(response.data);
+}
```

Comment:
Function has 2 responsibilities that aren't related. Setting state should be done at the component (or a hook) and instead return `response.data`